dev/devicelab/lib/tasks/gallery.dart

```diff
@@ -139,6 +139,22 @@ class GalleryTransitionTest {
         'worst_frame_rasterizer_time_millis',
         '90th_percentile_frame_rasterizer_time_millis',
         '99th_percentile_frame_rasterizer_time_millis',
+        'average_layer_cache_count',
```

Comment:
@keyonghan @dnfield is it sufficient to add these new metrics here for them to show up in SkiaPerf, or do you know if these metrics also have to be registered somewhere else as well?

Reply:
This should be sufficient, but this particular one only affects gallery. Is there some reason to not just add these to the default keys in perf_test.dart?

Reply:
Yeah, we upload whatever is included in the test result, and Skia perf simply picks up everything in GCS.

Reply:
Is there a reason that the vsync keys are not added to the CommonScoreKeys there?

Reply:
Probably whoever did it didn't realize they could or was only looking at gallery.

Reply:
I'm not talking about gallery, there seems to be inconsistent use of that variable. The vsync keys are added manually in gallery and they are also added manually in the general perf test wrapper, but they are there in all timeline summaries. There seems to be inconsistent use of these keys.

Also, kCommonScoreKeys is only the default for the e2e tests which do not produce these values. The non-e2e tests are the only tests where these keys can be found and there is no default list for those benchmarks.

There is a lot of churn in this part of the code with new benchmark variations being added and too many variant perf benchmarks that try to share this code to know where to add these keys responsibly. This is likely a cleanup task outside the scope of this specific change.

Reply:
SGTM - would you mind adding a TODO and filing a bug?

Reply:
Filed https://github.com/flutter/flutter/issues/89402 for the lack of these metrics in the e2e data stream.

Reply:
Filed a general "key specification is a mess" issue under https://github.com/flutter/flutter/issues/89403

Reply:
As part of the recent commit to publish the cache metrics from FrameTimings for the e2e benchmarks, I added the new keys to the default keys so now any benchmark that doesn't customize its keys will have these metrics tracked in skia perf...